bpf/lib/policy.h

```diff
@@ -133,11 +144,15 @@ policy_can_access_ingress(struct __sk_buff *skb, __u32 src_identity,
 	if (ret >= TC_ACT_OK)
 		return ret;
 
-	// cidr_addr_size is a compile time constant so this should all be inlined neatly.
-	if (cidr_addr_size == sizeof(union v6addr) && lpm6_ingress_lookup(cidr_addr))
-		goto allow;
-	if (cidr_addr_size == sizeof(__be32) && lpm4_ingress_lookup(*(__be32 *)cidr_addr))
-		goto allow;
+	/* CIDR policy only applies to traffic peering with something that is
+	 * not managed by Cilium. */
```

Comment:
Nit: `CIDR policy only applies to traffic peering with something that is not managed by Cilium. This translates to all the reserved identities.`

Reply:
Good suggestions. I'm going to merge this so we can backport and tag the release today as that will take another couple of hours. Let's get these recommendations in in a follow-up PR.